adoc/MAIN.release-notes.adoc

```diff
@@ -171,7 +173,60 @@ The `caaspctl` tool has been renamed from its working title to
 * https://bugzilla.suse.com/show_bug.cgi?id=1145568 - [remove-node] failed disarming kubelet due to 63 characters limitation
 * https://bugzilla.suse.com/show_bug.cgi?id=1145718 - Services do not promote their logs with the right priority to journald
 * https://bugzilla.suse.com/show_bug.cgi?id=1145773 - Server can't halt/reboot when Ceph RBD volume are mapped
-* https://bugzilla.suse.com/show_bug.cgi?id=1145904 - Some kured pods are in CrashLoopBackOff because they can't connect to apiserver. Fix is coming with next release. 
+* https://bugzilla.suse.com/show_bug.cgi?id=1145904 - Some kured pods are in CrashLoopBackOff because they can't connect to apiserver. Fix is coming with next release.
+
+== Support and Life Cycle
+
+{productname} is backed by award-winning support from SUSE, an established technology leader with a proven history of delivering enterprise-quality support services.
+
+{productname} {productmajor} has a two-year life cycle. Each version will receive updates while it is current, and will be subject to critical updates for the remainder of its life cycle.
+
+For more information, check our Support Policy page https://www.suse.com/support/policy.html.
+
+== Support Statement for {productname} {LINK]REPORT BUG#
+To receive support, you need an appropriate subscription with SUSE. For more information, see https://www.suse.com/support/programs/subscriptions/?id=SUSE_CaaS_Platform.
+
+The following definitions apply:
+
+L1
+Problem determination, which means technical support designed to provide compatibility information, usage support, ongoing maintenance, information gathering and basic troubleshooting using available documentation.
+
+L2
+Problem isolation, which means technical support designed to analyze data, reproduce customer problems, isolate problem area and provide a resolution for problems not resolved by Level 1 or prepare for Level 3.
+
+L3
+Problem resolution, which means technical support designed to resolve problems by engaging engineering to resolve product defects which have been identified by Level 2 Support.
+
+For contracted customers and partners, {productname} {productmajor} is delivered with L3 support for all packages, except for the following:
+
+Technology Previews
+
+Packages that require an additional customer contract
+
+Packages with names ending in -devel (containing header files and similar developer resources) will only be supported together with their main packages.
+
+SUSE will only support the usage of original packages. That is, packages that are unchanged and not recompiled.
+
+== Documentation and Other Information
+
+=== Available on the Product Media 
```

Comment:
* Afaiu, there is no actual media, people just install another repo (?). If so, some parts of this section do not really make sense (i.e. `Read the READMEs on the media`, `Find more information in the docu directory`). Package changelogs of course still exist.

* Version with nicer formatting available as part of https://gitlab.nue.suse.com/documentation/release-notes-sles/raw/12_SP5/adoc/about-product.adoc.

Reply:
For now, I've commented out what I think is irrelevant. @qnetter could you please review this part? Is it a good idea to for example point to the CLI `--help` option of `skuba`?